source/_posts/2017-02-03-hassbian-toybox.markdown

```diff
@@ -0,0 +1,62 @@
+---
+layout: post
+title: "HASSbian 1.1 - The Toy-box"
+description: "New release of the HASSbian Raspberry Pi Image for Home Assistant"
+date: 2017-02-04 05:00:00 -0400
+date_formatted: "February 4, 2017"
+author: Fredrik Lindqvist
+comments: true
+categories: Technology
+---
+
+Tonight I'm happy to announce a new release of the our Raspberry Pi image, **HASSbian 1.1 - The Toy-box.**  
+Why Toy-box you wonder? Because it encompass the changes pretty well.
+
+Changes from previous image are big and small but lets start with the interesting things.
+
+### {% linkable_title Hassbian-scrips %}
+A set of script written to add extra functionality to your raspberry pi installation. 
+This scripts are run as the `pi` user and installs a set of tools or packages.
+Currently includes:
+ - Install Libcec. Adds local [HDMI CEC support][cec].
+ - Install Mossquitto. Installs the latest Mosquitto package and client tools from the Mosquitto projects official repository. Now includes web-socket support.
+ - Install OpenZWave. Installs OpenZWave and prepares for using a USB or GPIO ZWave controller.
+ - Install Samba. Installs the Samba packages and shares you configuration over smb to be available to edit on any computer without the need for separate file transfer software. This share is unsecured and it's usage is not recommended if your installation is publicly available.
+
+All of these scripts are available in the directory `/home/pi/hassbian-scripts/`. This directory is actually a cloned git repository that's clone on first boot and can be updated to the latest release with ease after.
+To update the hassbian-scripts directory execute the following command as the `pi` user.
+```bash
+$ cd hassbian-scripts
+$ git pull
+```
+To use any of the hassbian-scripts, execute the following command as the `pi` user. This example uses the libcec script as an example.
```

Comment:
Two times example in one sentence. `The command below uses the libcec script as an example.`